conference/invoicing.py

```diff
@@ -51,12 +51,24 @@
 https://www.europython-society.org/
 """.strip()
 
+# EP2021: Not listing the VAT ID, since we're a non-profit and VAT exempt. 
+# The VAT ID would confuse people.
 EPS_21 = """
 Europython Society
 Ramnebacken 45
 424 38 Agnesberg
 Sweden
 Org ID: 802417-7704
+Contact Email: billing@europython.eu
+https://www.europython-society.org/
+""".strip()
+
+EPS_22 = """
```

Comment:
For EP2022, we will have to use the Irish VAT address and ID on the invoices. It's probably a good idea to add this in a comment.

Reply:
Added